src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -0,0 +1,31 @@
+const relativeTime = receivedTimestamp => {
+  const currentTime = Date.now();
+  const timeDifferenceMilliseconds = currentTime - receivedTimestamp;
+  const toMinuteDivisor = 1000 * 60;
+  const timeDifferenceMinutes = parseInt(
+    timeDifferenceMilliseconds / toMinuteDivisor,
+    10,
+  );
+
+  if (timeDifferenceMinutes > 10 * 60 || timeDifferenceMilliseconds < 0) {
+    return '';
```

Comment:
Any chance we can return null here? Personal preference but I think it handles nicer in rendering

Reply:
done 😄